app/build.gradle

```diff
@@ -90,20 +90,12 @@ android {
         staging {
             applicationId "com.nimbl3.staging"
         }
-
-        staging_internal {
-            applicationId "com.nimbl3.staging"
-            versionNameSuffix ".${project.versionNameSuffix}"
```

Comment:
Does this mean we can remove the following defined in the app gradle file? 

`project.ext.set("versionNameSuffix", "$gitCommitHash${ticketValue}")`